examples/flax/image-captioning/run_image_captioning_flax.py

```diff
@@ -389,6 +399,14 @@ def main():
     else:
         model_args, data_args, training_args = parser.parse_args_into_dataclasses()
 
+    if model_args.use_auth_token is not None:
+        warnings.warn(
+            "The `use_auth_token` argument is deprecated and will be removed in v5 of Transformers.", FutureWarning
```

Comment:
This is an example script, so it's not really linked to v5 of Transformers. Let's say it will be removed along the release of v4.34 (users can always use previous versions of the example scripts so it's not as bad as breaking change in the lib).